app/src/main/java/de/westnordost/streetcomplete/quests/leaf_detail/AddTreeLeafType.kt

```diff
@@ -0,0 +1,37 @@
+package de.westnordost.streetcomplete.quests.leaf_detail
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.elementfilter.toElementFilterExpression
+import de.westnordost.streetcomplete.data.osm.geometry.ElementGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.Element
+import de.westnordost.streetcomplete.data.osm.mapdata.MapDataWithGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.filter
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement.OUTDOORS
+import de.westnordost.streetcomplete.osm.Tags
+
+class AddTreeLeafType : OsmFilterQuestType<TreeLeafType>() {
+    override val elementFilter = """
+        nodes with
+          natural = tree
+          and !leaf_type
+          and !~"(taxon|genus|species).*"
+    """
+    override val changesetComment = "Specify leaf types"
+    override val wikiLink = "Key:leaf_type"
+    override val icon = R.drawable.ic_quest_leaf
+    override val isDeleteElementEnabled = true
+    override val achievements = listOf(OUTDOORS)
+    override val defaultDisabledMessage = R.string.default_disabled_msg_difficult_and_time_consuming
```

Comment:
Hmm, this is not entirely correct, but we have no text for "it's spammy". So, I guess, it's fine.

Reply:
Actual message is rather `default_disabled_msg_difficult_or_time_consuming` ("or" not "and") and "time consuming" seems good match for spammy.

And I would not add extra message just for that